corehq/form_processor/reprocess.py

```diff
@@ -30,6 +31,10 @@ class ReprocessingError(Exception):
 
 
 def reprocess_unfinished_stub(stub, save=True):
+    if any_migrations_in_progress(stub.domain):
+        logger.info("Ignoring stub during data migration: %s", stub.xform_id)
+        return
+
```

Comment:
What does this prevent?

Reply:
This prevents unifinished submission stubs from being re-processed.

Reply:
Cool, Thanks!